### ISSUE TITLE
Instant crash when setting up Osmand Cloud, can't backup anymore

### ISSUE THREAD

**Issue by Guylby** [osmandapp/OsmAnd#19742]
### Description

I tried to setup Osmand cloud on the android app, put my email adress then the confirmation code and got an immediate crash.
Now entering the Backup & Restore Menu causes an immediate crash, I can't even do offline backup, as exporting a Profile will also result in a crash:

5 crash event log are joined below
The first 4 are from the Osmand Cloud setup and entering the Backup & Restore menu
The last one is from going into a profile, selecting "Export Profile", selecting all settings, clicking "Continue" and getting a crash immediately. 

### Steps to reproduce

See above

### Actual result

Osmand crash with Osmand Cloud

### Expected result

Osmand doesn't crash with Osmand Cloud

### Your Environment (required)

**WARNING** Crash-Logs **MAY** contain information you deem sensitive.
Review this **CAREFULLY** before posting your issue!

```
OsmAnd Version: 4.7.13
Android/iOS version: Android 13
Device model: Xperia 10 iii

```

<details>
    <summary>Crash-Logs: </summary>

```
Version  OsmAnd+ 4.7.13
02.05.2024 3:41:57
Apk Version : 4.7.13 4713
Exception occurred in thread Thread[AsyncTask #4,5,main] : 
java.lang.RuntimeException: An error occurred while executing doInBackground()
	at android.os.AsyncTask$4.done(AsyncTask.java:415)
	at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:381)
	at java.util.concurrent.FutureTask.setException(FutureTask.java:250)
	at java.util.concurrent.FutureTask.run(FutureTask.java:269)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:644)
	at java.lang.Thread.run(Thread.java:1012)
Caused by: java.lang.ClassCastException: java.lang.Double cannot be cast to java.lang.Float
	at net.osmand.plus.settings.backend.backup.GpxAppearanceInfo.<init>(GpxAppearanceInfo.java:87)
	at net.osmand.plus.settings.backend.backup.items.GpxSettingsItem.createGpxAppearanceInfo(GpxSettingsItem.java:137)
	at net.osmand.plus.settings.backend.backup.items.GpxSettingsItem.<init>(GpxSettingsItem.java:46)
	at net.osmand.plus.settings.backend.backup.SettingsHelper.prepareSettingsItems(SettingsHelper.java:190)
	at net.osmand.plus.settings.backend.backup.SettingsHelper.getFilteredSettingsItems(SettingsHelper.java:128)
	at net.osmand.plus.settings.backend.backup.SettingsHelper.getFilteredSettingsItems(SettingsHelper.java:117)
	at net.osmand.plus.backup.BackupHelper$4.getLocalItems(BackupHelper.java:797)
	at net.osmand.plus.backup.BackupHelper$4.doInBackground(BackupHelper.java:694)
	at net.osmand.plus.backup.BackupHelper$4.doInBackground(BackupHelper.java:678)
	at android.os.AsyncTask$3.call(AsyncTask.java:394)
	at java.util.concurrent.FutureTask.run(FutureTask.java:264)
	... 3 more
Version  OsmAnd+ 4.7.13
02.05.2024 3:55:21
Apk Version : 4.7.13 4713
Exception occurred in thread Thread[AsyncTask #3,5,main] : 
java.lang.RuntimeException: An error occurred while executing doInBackground()
	at android.os.AsyncTask$4.done(AsyncTask.java:415)
	at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:381)
	at java.util.concurrent.FutureTask.setException(FutureTask.java:250)
	at java.util.concurrent.FutureTask.run(FutureTask.java:269)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:644)
	at java.lang.Thread.run(Thread.java:1012)
Caused by: java.lang.ClassCastException: java.lang.Double cannot be cast to java.lang.Float
	at net.osmand.plus.settings.backend.backup.GpxAppearanceInfo.<init>(GpxAppearanceInfo.java:87)
	at net.osmand.plus.settings.backend.backup.items.GpxSettingsItem.createGpxAppearanceInfo(GpxSettingsItem.java:137)
	at net.osmand.plus.settings.backend.backup.items.GpxSettingsItem.<init>(GpxSettingsItem.java:46)
	at net.osmand.plus.settings.backend.backup.SettingsHelper.prepareSettingsItems(SettingsHelper.java:190)
	at net.osmand.plus.settings.backend.backup.SettingsHelper.getFilteredSettingsItems(SettingsHelper.java:128)
	at net.osmand.plus.settings.backend.backup.SettingsHelper.getFilteredSettingsItems(SettingsHelper.java:117)
	at net.osmand.plus.backup.BackupHelper$4.getLocalItems(BackupHelper.java:797)
	at net.osmand.plus.backup.BackupHelper$4.doInBackground(BackupHelper.java:694)
	at net.osmand.plus.backup.BackupHelper$4.doInBackground(BackupHelper.java:678)
	at android.os.AsyncTask$3.call(AsyncTask.java:394)
	at java.util.concurrent.FutureTask.run(FutureTask.java:264)
	... 3 more
Version  OsmAnd+ 4.7.13
02.05.2024 3:56:06
Apk Version : 4.7.13 4713
Exception occurred in thread Thread[AsyncTask #1,5,main] : 
java.lang.RuntimeException: An error occurred while executing doInBackground()
	at android.os.AsyncTask$4.done(AsyncTask.java:415)
	at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:381)
	at java.util.concurrent.FutureTask.setException(FutureTask.java:250)
	at java.util.concurrent.FutureTask.run(FutureTask.java:269)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:644)
	at java.lang.Thread.run(Thread.java:1012)
Caused by: java.lang.ClassCastException: java.lang.Double cannot be cast to java.lang.Float
	at net.osmand.plus.settings.backend.backup.GpxAppearanceInfo.<init>(GpxAppearanceInfo.java:87)
	at net.osmand.plus.settings.backend.backup.items.GpxSettingsItem.createGpxAppearanceInfo(GpxSettingsItem.java:137)
	at net.osmand.plus.settings.backend.backup.items.GpxSettingsItem.<init>(GpxSettingsItem.java:46)
	at net.osmand.plus.settings.backend.backup.SettingsHelper.prepareSettingsItems(SettingsHelper.java:190)
	at net.osmand.plus.settings.backend.backup.SettingsHelper.getFilteredSettingsItems(SettingsHelper.java:128)
	at net.osmand.plus.settings.backend.backup.SettingsHelper.getFilteredSettingsItems(SettingsHelper.java:117)
	at net.osmand.plus.backup.BackupHelper$4.getLocalItems(BackupHelper.java:797)
	at net.osmand.plus.backup.BackupHelper$4.doInBackground(BackupHelper.java:694)
	at net.osmand.plus.backup.BackupHelper$4.doInBackground(BackupHelper.java:678)
	at android.os.AsyncTask$3.call(AsyncTask.java:394)
	at java.util.concurrent.FutureTask.run(FutureTask.java:264)
	... 3 more
Version  OsmAnd+ 4.7.13
02.05.2024 3:56:07
Apk Version : 4.7.13 4713
Exception occurred in thread Thread[main,5,main] : 
java.lang.ClassCastException: java.lang.Double cannot be cast to java.lang.Float
	at net.osmand.plus.settings.backend.backup.GpxAppearanceInfo.<init>(GpxAppearanceInfo.java:87)
	at net.osmand.plus.settings.backend.backup.items.GpxSettingsItem.lambda$createGpxAppearanceInfo$0$net-osmand-plus-settings-backend-backup-items-GpxSettingsItem(GpxSettingsItem.java:135)
	at net.osmand.plus.settings.backend.backup.items.GpxSettingsItem$$ExternalSyntheticLambda0.onGpxDataItemReady(Unknown Source:2)
	at net.osmand.plus.track.helpers.GpxDbHelper.onProgressUpdate(GpxDbHelper.java:317)
	at net.osmand.plus.track.helpers.GpxReaderTask.onProgressUpdate(GpxReaderTask.java:199)
	at net.osmand.plus.track.helpers.GpxReaderTask.onProgressUpdate(GpxReaderTask.java:43)
	at android.os.AsyncTask$InternalHandler.handleMessage(AsyncTask.java:791)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loopOnce(Looper.java:346)
	at android.os.Looper.loop(Looper.java:475)
	at android.app.ActivityThread.main(ActivityThread.java:7950)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:942)
```
```
Version  OsmAnd+ 4.7.13
02.05.2024 4:06:59
Apk Version : 4.7.13 4713
Exception occurred in thread Thread[main,5,main] : 
java.lang.ClassCastException: java.lang.Double cannot be cast to java.lang.Float
	at net.osmand.plus.settings.backend.backup.GpxAppearanceInfo.<init>(GpxAppearanceInfo.java:87)
	at net.osmand.plus.settings.backend.backup.items.GpxSettingsItem.createGpxAppearanceInfo(GpxSettingsItem.java:137)
	at net.osmand.plus.settings.backend.backup.items.GpxSettingsItem.<init>(GpxSettingsItem.java:46)
	at net.osmand.plus.settings.backend.backup.SettingsHelper.prepareSettingsItems(SettingsHelper.java:190)
	at net.osmand.plus.settings.fragments.ExportSettingsFragment.prepareFile(ExportSettingsFragment.java:219)
	at net.osmand.plus.settings.fragments.ExportSettingsFragment.onContinueButtonClickAction(ExportSettingsFragment.java:157)
	at net.osmand.plus.settings.fragments.BaseSettingsListFragment.lambda$onCreateView$0$net-osmand-plus-settings-fragments-BaseSettingsListFragment(BaseSettingsListFragment.java:113)
	at net.osmand.plus.settings.fragments.BaseSettingsListFragment$$ExternalSyntheticLambda2.onClick(Unknown Source:2)
	at android.view.View.performClick(View.java:7516)
	at android.view.View.performClickInternal(View.java:7490)
	at android.view.View.-$$Nest$mperformClickInternal(Unknown Source:0)
	at android.view.View$PerformClick.run(View.java:29349)
	at android.os.Handler.handleCallback(Handler.java:942)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loopOnce(Looper.java:346)
	at android.os.Looper.loop(Looper.java:475)
	at android.app.ActivityThread.main(ActivityThread.java:7950)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:942)
```

</details>

**Comment by myfta** [comment]
I can confirm the same instant crash. I have submitted the crash report.

**Comment by Guylby** [comment]
Issue is fixed for me in 4.7.14, thanks!
